examples/thirdPartApp/overlays/tests/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+configMapGenerator:
+  - name: example-app-config
+    literals:
+      - APP_ENV=testing
+secretGenerator:
+  - name: example-app-secrets
+    literals:
+      - DB_PASSWORD=password
+patchesStrategicMerge:
```

Comment:
I think `patchesStrategicMerge` is now deprecated. 
Could you use [patches](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/) syntax?

Reply:
Done